jupyter-extension/src/plugins/mount/mountDrive.ts

```diff
@@ -115,6 +109,8 @@ export class MountDrive implements Contents.IDrive {
     localPath: string,
     options?: Contents.IFetchOptions,
   ): Promise<Contents.IModel> {
+    this.setupScrollingHandler();
```

Comment:
I just moved this call to be the first line of this method since it can happen independently from everything else.